iree/compiler/Dialect/VM/Target/C/CModuleTarget.cpp

```diff
@@ -135,8 +177,19 @@ static LogicalResult initializeGlobals(IREE::VM::ModuleOp moduleOp,
              << "Initializers for globals not supported yet";
     }
   }
+  // TODO(simon-camp): Support globals with different element type
 
-  // TODO(simon-camp): Support vm.global.i64 and vm.global.ref
+  for (auto rodataOp : moduleOp.getOps<IREE::VM::RodataOp>()) {
+    std::string buffer_name =
+        moduleOp.getName().str() + "_" + rodataOp.getName().str();
+    output << "iree_vm_buffer_initialize("
+           << "IREE_VM_BUFFER_ACCESS_ORIGIN_HOST, "
```

Comment:
this should be `IREE_VM_BUFFER_ACCESS_ORIGIN_MODULE` - that indicates that the buffer will only be available for as long as the module is loaded/initialized